src/components/dictionaryConcepts/components/ExternalMapping.jsx

```diff
@@ -0,0 +1,93 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { FormGroup, Input } from 'reactstrap';
+
+class ExternalMapping extends Component {
+  handleChange = (event) => {
+    this.props.handleChange(event);
+  };
+
+  render() {
+    const {
+      map_type,
+      term,
+      source_url,
+      code,
+    } = this.props;
+
+    return (
+      <div>
+        <form>
+          <div>
+            <FormGroup style={{ marginTop: '12px' }}>
```

Comment:
@Duncanian Why are you putting styles here?

Reply:
@muhozi , let me refactor.